portingdb/tables.py

```diff
@@ -139,6 +139,12 @@ def nonblocking(self):
     def list_tracking_bugs(self):
         return [tb.url for cp in self.collection_packages for tb in cp.tracking_bugs]
 
+    @property
+    def last_link_update(self):
+        values = [link.last_update for cp in self.collection_packages
+                for link in cp.links if link.last_update is not None]
+        return values[0] if values else None
```

Comment:
Nitpick: I don't think this is one of the rare cases where the if-else operator is better than a full if statement.